datalad_deprecated/resources/website/assets/js/main.js

```diff
@@ -565,14 +631,41 @@ function directory(jQuery, md5) {
         else if (traverse.type === 'search')
           window.location.search = traverse.next;
       });
+
+      jQuery('table button').click(function(event) {
+        event.stopPropagation();
+        if (!jQuery(this).next().is(":visible")) {
+          jQuery('.context-content:visible').hide();
+          jQuery(this).next().show();
+        }
+        else {
+          jQuery('.context-content:visible').hide();
+        }
+      });
+      // Prevent double opening on middle/ctrl click
+      jQuery('table a, .separator').click(function(event) {
+        event.stopPropagation();
+      });
+      jQuery('.copy').click(function(event) {
+        event.stopPropagation();
+        copyToClipboard(jQuery(this).parent().parent().next().prop('href'));
+        jQuery('.context-content:visible').hide();
+      });
+      jQuery(window).click(function(event) {
+        jQuery('.context-content:visible').hide();
+      });
+
       // add visit folder button
       var crumbs = bread2crumbs(jQuery, md5)
       var curdir = jQuery(jQuery.parseHTML(crumbs[crumbs.length-1])).attr("href");
       if (curdir.indexOf("/?dir=") > -1){
         curdir = (curdir.slice(0, curdir.indexOf("/?dir=")) + curdir.slice(curdir.indexOf("/?dir=") + 6));
       }
+
+      var rawCrumbs = loc().href.replace(/#.*/, '').replace(/\/$/, '').split('/');
+      
       jQuery('#directory_filter').prepend('<a id="folder-link" href="'+curdir+'">'+
-                                          '<span class="visit-folder">[Visit folder]</span>'+
+                                          '<span class="visit-folder ' + getNodeType(jQuery, md5, rawCrumbs.join('/')) + '">[Open Folder]</span>'+
```

Comment:
note: `visit-folder` custom css was removed, so class has no effect here, but I am ok to keep it around